Installation.md

```diff
@@ -0,0 +1,54 @@
+# Installation
+## Run on local machine
+
```

Comment:
You should rather use a virtual environment, especially when running on your local machine. I.e.:

    python3 -m venv .pyenv
    . .pyenv/bin/activate

This allows you to install python packages locally and `python` points to the right Python version (i.e., you do not have to write `python3` explicitly).